### PR TITLE
A script that can create arbitrary amounts of s3 data for benchmarks

### PR DESCRIPTION
**Pull request by quodlibetor**
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/5503)

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -4141,6 +4219,17 @@ dependencies = [
...
+[[package]]
+name = "time"
+version = "0.1.44"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "6db9e6914ab8b1ae1c260a4ae7a49b6c5611b40328a735b21862567685e73255"
+dependencies = [
+ "libc",
+ "wasi 0.10.0+wasi-snapshot-preview1",
+ "winapi",
+]
```
Comment (benesch) [inline Cargo.lock:4231]: Ahhhh noo! Back from the dead! Not sure what's bringing this in, but let's find it and squash it.

File: test/performance/s3-datagen/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+rusoto_credential = { git = "https://github.com/rusoto/rusoto.git" }
+rusoto_s3 = { git = "https://github.com/rusoto/rusoto.git" }
+structopt = "0.3.21"
+tokio = { version = "1.0.0", features = ["macros", "net", "rt", "time"] }
+tracing = "0.1.22"
+tracing-subscriber = "0.2.15"
+anyhow = "1.0.38"
+bytefmt = "0.1.7"
+indicatif = "0.15.0"
+futures = "0.3.12"
```
Comment (benesch) [inline test/performance/s3-datagen/Cargo.toml:20]: Since you're going back through CI, `cargo manifmt`, or just sort these!

**Review by benesch — APPROVED** [review]
LGTM if we can squash this time v0.1 dependency!

File: Cargo.lock (hunk 1 above)
Comment (quodlibetor) [reply Cargo.lock]: it was tracing-subscriber.

**Comment by benesch** [comment]
I think you're missing a `publish = false`. That'll suppress the license check.

**Pull request merged**